nosekatreport/__init__.py

```diff
@@ -1,2 +1,19 @@
+# Copyright (c) 2013 National Research Foundation (South African Radio Astronomy Observatory)
```

Comment:
The year must match when the file was added.  These files were only added in 2017.